src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.tsx

```diff
@@ -22,11 +22,11 @@ const MSG = defineMessages({
   },
   roleDescription2: {
     id: `dashboard.PermissionManagementDialog.PermissionManagementCheckbox.roleDescription2`,
-    defaultMessage: 'Coming soon...',
+    defaultMessage: `Arbitration allows you to resolve disputes, make state changes, and punish bad behavior.`,
```

Comment:
Mostly just a curiosity, is American English preferred? e.g. behavior vs behaviour. I've noticed both used in the Dapp.